spec/factories/events.rb

```diff
@@ -1,8 +1,10 @@
 FactoryGirl.define do
   factory :event do
-    activity {FactoryGirl.create(:activity)}
-    location {Faker::Address.country}
-    time {Faker::Date.between(Date.today, 10.days.from_now)}
+    activity { FactoryGirl.create(:activity) }
```

Comment:
If the attribute is named the same as a model, you can just do `activity` and it'll automatically grab the factory.